components/widgets/inputs/users_emails_input.tsx

```diff
@@ -276,6 +276,12 @@ export default class UsersEmailsInput extends React.PureComponent<Props, State>
             if (this.props.onUsersLoad) {
                 this.props.onUsersLoad(options);
             }
+            const optionWithLabel = options.forEach((option) => {
```

Comment:
I've tested the changes with a screen reader and I still get undefined for the focused option.
I've tried out setting the option labels before setting state and it seems to work properly.

Reply:
Ditto what the others said. Adding a return value isn't necessary. Only setting label for the values that are sent to `callback` is needed. e.g. this works for me locally:

![change](https://user-images.githubusercontent.com/13738432/196769932-c2c0b1d8-a1e3-439b-94f6-6de54ace1dab.png)